server/src/main.rs

```diff
@@ -161,59 +166,146 @@ fn mint_service_error<E>(e: E) -> SdkError<E> {
     )
 }
 
-#[tokio::main]
-async fn main() -> Result<(), Error> {
-    tracing_subscriber::fmt()
-        .with_env_filter(EnvFilter::from_default_env())
-        .without_time(/* cloudwatch does that */).init();
+/// Seed the database.
+///
+/// This will register a test event (with id `00000000000000000000000000`) and
+/// a number of questions for it in the database, whether it's an in-memory [`Local`]
+/// database or a local instance of DynamoDB. Note that in the latter case
+/// we are checking if the test event is already there, and - if so - we are _not_ seeding
+/// the questions. This is to avoid creating duplicated questions when re-running the app.
+/// And this is not an issue of course when running against our in-memory [`Local`] database.
+///
+/// The returned vector contains IDs of the questions related to the test event.
+#[cfg(debug_assertions)]
+async fn seed(backend: &mut Backend) -> Vec<Ulid> {
+    #[derive(serde::Deserialize)]
+    struct LiveAskQuestion {
+        likes: usize,
+        text: String,
+        hidden: bool,
+        answered: bool,
+        #[serde(rename = "createTimeUnix")]
+        created: usize,
+    }
 
-    #[cfg(not(debug_assertions))]
-    let backend = Backend::dynamo().await;
-    #[cfg(debug_assertions)]
-    let backend = if std::env::var_os("USE_DYNAMODB").is_some() {
-        Backend::dynamo().await
-    } else {
-        use rand::prelude::SliceRandom;
-        use serde::Deserialize;
-        use std::time::Duration;
-
-        #[cfg(debug_assertions)]
-        #[derive(Deserialize)]
-        struct LiveAskQuestion {
-            likes: usize,
-            text: String,
-            hidden: bool,
-            answered: bool,
-            #[serde(rename = "createTimeUnix")]
-            created: usize,
-        }
+    let seed: Vec<LiveAskQuestion> = serde_json::from_str(SEED).unwrap();
+    let seed_e = Ulid::from_string("00000000000000000000000000").unwrap();
+    let seed_e_secret = "secret";
+
+    match backend {
```

Comment:
Ah, so, this isn't quite what I had in mind. You shouldn't need this match now. at least not for the calls to `event` and `ask`. You should only need it for the small section after seeding all the questions that sets the vote count + answered/hidden flags.